app/grants/views.py

```diff
@@ -97,11 +98,11 @@
 
 kudos_reward_pks = [12580, 12584, 12572, 125868, 12552, 12556, 12557, 125677, 12550, 12392, 12307, 12343, 12156, 12164]
 
-if not clr_active:
```

Comment:
all of these variables (line 77 onward) need to be driven by the DB now, not just commented out/ignored

Reply:
They are commented out cause I'd like you to check out how we'd wanna handle this ! I'm not very clear on what is it we wanna do with this piece of code !

For example: 
How we do we handle diff routes when clr is not all active / when a few are active / etc